### PR TITLE
FDN-1843: add NODE v20

### PR DESCRIPTION
**Pull request by himanshuupadhyay101**

### CODE REVIEW

**Review by RoryH — APPROVED** [review]
LGTM

Do we also need to think about updating the [JRE to v17 here](https://github.com/flowcommerce/docker/blob/main/Dockerfile-node#L19C42-L19C42)?

This is just used for injecting the app env vars running the services via `environment.jar`

File: Dockerfile-node [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN chown root:root /root/environment-provider*
 WORKDIR /root
 
 RUN apk update && apk upgrade && \
-    apk add --no-cache openjdk11-jre && \
+    apk add --no-cache eclipse-temurin:17-jre && \
```
Comment (RoryH) [inline Dockerfile-node:19]: This does not seem to be an Alpine package, we should probably use one of these: https://pkgs.alpinelinux.org/packages?name=openjdk17-jre&branch=edge&repo=&arch=&maintainer=

Comment (himanshuupadhyay101) [reply Dockerfile-node]: just updated to openjdk17-jre

**Review by RoryH — APPROVED** [review]
LGTM

**Pull request merged**